modules/contracts/adapters/cw-staking/src/handlers/query.rs

```diff
@@ -30,7 +30,14 @@ pub fn query_handler(
                 // the query can be executed on the local chain
                 let mut provider = resolver::resolve_local_provider(&local_provider_name)
                     .map_err(|e| StdError::generic_err(e.to_string()))?;
-                provider.fetch_data(deps, env, None, ans_host, abstract_registry, staking_token)?;
+                provider.fetch_data(
+                    deps,
+                    env,
+                    None,
+                    ans_host,
+                    &abstract_registry,
```

Comment:
Is there any implementation that actually uses the version control address provided here? (should probably rename this "registry" to version_control_contract).

Reply:
Osmosis uses it to get proxy addr